content/resources.md

```diff
@@ -111,11 +111,10 @@ Materials relating to …. well… malware reversing, forensics, threat intellig
 
 **CDC Materials**
 
-Materials for learning Cyber Defense Competition skills.
-Join us and learn about CDC in the #cdc channel by adding the CDC role in Discord!
+Materials for learning Cyber Defense Competition Skills. 
```

Comment:
pls revert both of these lines:
- skills doesnt need to be caps
- that discord is currently not a good place for resources and we need to rethink how we're linking our discords anyhow